lib/patches.js

```diff
@@ -18,6 +18,13 @@ module.exports = function(Sequelize) {
     // define patches
     return semverSelect.object(sequelizeVersion, {
         /*
+         * Patches underscoredIf location differing in v2
+         */
+        underscoredIf: {
+          '^2.0.0': Sequelize.Utils._.underscoredIf,
+          '* || >=4.0.0-1': Sequelize.Utils.underscoredIf
```

Comment:
`'* || >=4.0.0-1'` should be `'>=3.0.0 || ^4.0.0-0'` to match the rest (and so not to match v1.x.x.